documentation/configuringDynawo/DynawoInputFiles.tex

```diff
@@ -679,19 +679,32 @@ \subsubsection{Define sets of static references}
 
 \subsection{par file}
 
-\subsubsection{Set a reference to a value stored in the iidm file}
+\subsubsection{Set a reference to a value stored in the iidm file or a given par file}
 \label{sec:Set a reference to a value stored in the iidm file}
 
 If the parameter's value is stored in the iidm file, a reference to this value can be added in the par file as in the example bellow:
-\begin{lstlisting}[language=XML, morekeywords={reference}]
+\begin{lstlisting}[language=XML, morekeywords={iidmreference}]
 <set>
   <par type="ParType" name="ParName" value="ParValue"/>
   ...
   <reference type="RefType" name="RefName" origData="IIDM" origName="NameIIDM" componentId="Model_id_in_iidm_File"/>
 </set>
 \end{lstlisting}
 
-When no componentId is given, it is assumed that the data is linked with the static component which id is given by the staticId defined in the dyd file.
+When no componentId is given, it is assumed that the data is linked with the static component which id is given by the staticId defined in the dyd file.  \\
+
+If the parameter's value is stored in the par file, a reference to this value can be added in the par file as in the example below:
```

Comment:
I would propose "To use the value from another parameter, a parameter reference can be used as shown in the example below:"